app/models/book.rb

```diff
@@ -2,14 +2,20 @@ class Book < ApplicationRecord
   has_many :checkouts
   has_many :users, through: :checkouts
 
-  validates :title, presence: true, uniqueness: { case_sensitive: false}
+  validates :title, presence: true, uniqueness: { case_sensitive: false }
   validates :author, presence: true
 
-  def self.available
-    includes(:checkouts).where(checkouts: { id: nil })
+  def self.unavailable
+    Book.joins(:checkouts).where(checkouts: {closed_at: nil })
   end
 
-  def self.unavailable
-    includes(:checkouts).where.not(checkouts: { id: nil })
+  def self.available
+    Book.joins(
```

Comment:
I was not quite sure how to convert this to Rails 5. The things I tried did not work.